Url Scan - rate limit (#8908)
demisto/content: Packs/UrlScan/ReleaseNotes/1_0_2.md

#### Integrations
##### urlscan.io
- Improved error message when rate limit has been breached.

> - Added warning when rate limit is reached to 10.

Reviewer: ```suggestion
- Added warning when the rate limit is 10 or less.
```
